api/mapping/forms.py

```diff
@@ -20,8 +22,22 @@ class ScanReportForm(forms.Form):
         label="WhiteRabbit ScanReport",
         widget=forms.FileInput(attrs={"class": "form-control"}),
     )
-
-
+    def clean_scan_report_file(self):
+        xlsx = Xlsx2csv(self.cleaned_data['scan_report_file'], outputencoding="utf-8")
+
+        filepath = "/tmp/{}.csv".format(xlsx.workbook.sheets[0]["name"])
+        xlsx.convert(filepath)
+
+        with open(filepath, "rt") as f:
+            reader = csv.reader(f)
+            csv_header=next(reader)  # Get header row
+            set_header=['Table', 'Field', 'Description', 'Type', 'Max length', 'N rows', 'N rows checked', 'Fraction empty', 'N unique values', 'Fraction unique', 'Flag', 'Classification']
+            if set(set_header)==set(csv_header):
+                return self.cleaned_data['scan_report_file']
+            else:
+                raise (forms.ValidationError("Please check the column names in your Scan Report"))
```

Comment:
Can you add the column names in the ValidationError message, otherwise the user doesnt know what is missing. Example: "Please check the followings columns exist in the Scan Report? Column names: Table, Field, Description, Type, Max length, N rows, N rows checked, Fraction empty, N unique values, Fraction unique, Flag, Classification."